.ci/validate-examples.py

```diff
@@ -24,7 +24,7 @@
 FILENAME_REGEX = r"^[0-9A-Za-z\-\.]+$"
 SATURN_DIR_NAME = ".saturn"
 SATURN_JSON_NAME = "saturn.json"
-SATURN_JSON_KEYS = ["image", "jupyter", "environment_variables"]
+SATURN_JSON_KEYS = ["image", "jupyter", "environment_variables", "description"]
```

Comment:
This is unrelated, but I noticed it and think this is better.

Reply:
oh yeah works for me!